bitnami/rabbitmq/Chart.yaml

```diff
@@ -23,4 +23,4 @@ name: rabbitmq
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/rabbitmq
   - https://www.rabbitmq.com
-version: 11.1.5
+version: 11.1.6
```

Comment:
Hi @DSczyrba ,

Thanks for the contribution! I think we are adding a feature here so the version should bump a minor (`11.2.0`)